packages/tc-ui/src/pages/delete/__tests__/e2e-delete-record.cyp.js

```diff
@@ -1,32 +1,22 @@
-const {
-	code,
-	deleteConfirmText,
-} = require('../../../test-helpers/mainTypeData.json');
-const {
-	createType,
-	visitEditPage,
-	visitMainTypePage,
-	pickChild,
-	save,
-} = require('../../../test-helpers/cypress');
```

Comment:
Highlight #12 These shared helpers wrapping collections of cypress commands are completely gone, replaced with a mixture of a handful of tailored helpers in each spec file to take care of the least DRY things, but mostly just done away with helpers, preferring to use cypress methods directly in tests